missions/as_traidor.sqf

```diff
@@ -166,7 +166,7 @@ else
 		};
 	};
 
-_nul = [1200,_tsk] spawn borrarTask;
+_nul = [28800,_tsk] spawn borrarTask;

```

Comment:
Er, actually, I read what the forum post said, but Borrartask seems to be a timeout for how long the mission stays active, not the timeout *between* missions.